src/Umbraco.Web.UI.Client/src/views/documenttypes/views/permissions/permissions.controller.js

```diff
@@ -6,10 +6,10 @@
  * @description
  * The controller for the content type editor property dialog
  */
-(function() {
+(function () {
     'use strict';
 
-    function PermissionsController($scope, $timeout, contentTypeResource, iconHelper, contentTypeHelper, localizationService, overlayService) {
+    function PermissionsController($scope, $timeout, contentTypeResource, iconHelper, contentTypeHelper, localizationService, overlayService, editorService) {
```

Comment:
I should remove localizationService and overlayService here as they aren't used anymore

Reply:
Hey @patrickdemooij9, this is a really nice change, it's a much, much improved UX, worthy of a double thumbs up 👍 👍  

I removed those params and did a quick syntax update in both files (using es6 methods instead of underscore, single quotes, arrows).

Reply:
Ah great, thank you! I totally forgot that I had left that comment here 😅